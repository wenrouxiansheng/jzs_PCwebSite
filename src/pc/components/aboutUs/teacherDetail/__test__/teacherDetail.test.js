import React from "react";
import renderer from "react-test-renderer";
import TeacherDetail from "../index";

describe("PC:<TeacherDetail />", () => {
  it("render", () => {
    const wrapper = renderer.create(<TeacherDetail style={{ color: "red" }} />);
    let tree = wrapper.toJSON();

    //渲染测试
    expect(tree).toMatchSnapshot();
  });
});
