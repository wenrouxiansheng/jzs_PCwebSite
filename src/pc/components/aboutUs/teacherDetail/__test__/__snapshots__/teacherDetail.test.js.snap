// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PC:<TeacherDetail /> render 1`] = `
<div
  className="teacherDetail"
  style={
    Object {
      "color": "red",
    }
  }
>
  <img
    alt=""
  />
  <div
    className="info"
  >
    <h1>
      许杨
    </h1>
    <h3>
      个人简介
    </h3>
    <div>
      <span>
        教龄
        <span
          style={
            Object {
              "width": "100%",
            }
          }
        />
      </span>
      ：
      <div>
         
        童声、美声、民族、通俗唱法
      </div>
    </div>
    <div>
      <span>
        教学课程
        <span
          style={
            Object {
              "width": "100%",
            }
          }
        />
         
      </span>
      ：
      <div
        className="text"
      >
        9年
      </div>
    </div>
    <h3>
      个人荣誉
    </h3>
    <div
      className="honor"
      dangerouslySetInnerHTML={
        Object {
          "__html": "1989年6岁考入北京市少年宫独唱班学习童声唱法<br>
        1990年参加全国迎亚运少儿歌唱比赛获得二等奖<br>
        1995年考入中国交响乐团附属少年及女子合唱团并随团进行多次演出<br>
        2001年考入维也纳国立音乐与艺术大学 声乐 本硕连读<br>
        2010年加入维也纳国家歌剧院合唱团<br>
        在维也纳留学期间，曾多次参加大使馆组织的各项文艺活动，并受到大使接见；多次在奥地利、意大利、捷克等国家歌剧院进行演出；曾多次参加欧洲各国声乐比赛并获得优秀表演等奖项",
        }
      }
    />
  </div>
</div>
`;
